var/spack/repos/builtin/packages/mvdtool/package.py

```diff
@@ -16,6 +16,7 @@ class Mvdtool(CMakePackage):
     git      = "https://github.com/BlueBrain/MVDTool.git"
 
     version('develop', get_full_repo=False, submodules=True)
+    version('2.3.6', tag='v2.3.6', get_full_repo=False)
```

Comment:
@matz-e  Can we remove some of those old versions from the recipe ? Which are still needed ?

Reply:
I guess we could, there's always the possibility to go back in Spack history. Is there any harm leaving them in, though?

Reply:
No harm, I am just wondering because the list is getting long. We can also keep them in.

Reply:
Maybe only the latest `2.3`, `2.2` etc version can be kept? I'll leave it up to you if you want to remove them